portfolio/src/main/webapp/script.js

```diff
@@ -15,31 +15,23 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
-
 function addMessage() {
```

Comment:
It looks like you can add `// eslint-disable-line` after the method name to make the lint check ignore it

```
function addMessage() { // eslint-disable-line
```